extension.js

```diff
@@ -26,19 +29,33 @@ async function updateTheme(config, isDarkTheme = true) {
   await fse.writeFile(themePath, theme)
 }
 
-function promptForReload() {
+function checkThemeAgainstSettings() {
+  const darkConfig = workspace.getConfiguration("polychrome.dark")
+  const lightConfig = workspace.getConfiguration("polychrome.light")
+
+  if (
+    darkConfig.background !== darkTheme.polychromeConfig.background ||
+    darkConfig.primary !== darkTheme.polychromeConfig.primary ||
+    darkConfig.accent !== darkTheme.polychromeConfig.accent ||
+    lightConfig.background !== lightTheme.polychromeConfig.background ||
+    lightConfig.primary !== lightTheme.polychromeConfig.primary ||
+    lightConfig.accent !== lightTheme.polychromeConfig.accent
+  ) {
```

Comment:
I get the rationale for spot-checking, but as an outsider looking in I would've expected the reload to be prompted if the appearance differs _at all_ (due to bug fixes or changes in contrast for accessibility), via `_.deepEqual` or something.

Reply:
Hmmm... that's a good point. Maybe instead of saving the settings values, I could generate the theme and do that `deepEqual` comparison. Thanks for that suggestion @ericclemmons